config/sync/message.template.os2loop_message_answer_added.yml

```diff
@@ -9,6 +9,11 @@ label: 'Comment added to post'
 description: message_type
 text:
   -
-    value: 'Der er kommet et nyt svar til spørgsmålet: <a href="/node/[message:field-message-node-reference:nid]">[message:field-message-node-reference:field_description]</a>'
+    value: 'Der er kommet et nyt svar til spørgsmålet: <a href="/node/[message:os2loop_message_node_refer:entity:nid]">[message:os2loop_message_node_refer:entity:title]</a>'
```

Comment:
Should the message be written in English and translated to Danish?

Reply:
CHanged